client/src/CardComponent.css

```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
 
-body {
+.CardComponent body {
```

Comment:
kebab case for class names

Reply:
i think we are styling the entire class; tried kebab casing and i think it errors in this case?

Reply:
did you also kebab case the `CardComponent` class in `CardComponent.js`

Reply:
that's what i tried to do but when i tried to do Card-Component in CardComponent.js that's where it was errorring.